.github/workflows/ci.yml

```diff
@@ -0,0 +1,141 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+name: Build flink-connector-elasticsearch
+on: [push, pull_request]
+jobs:
+  compile_ci_jdk8:
+    runs-on: ubuntu-latest
+    env:
+      MVN_GOALS: clean install
+      MVN_PROFILES: -Pcheck-convergence -Dscala-2.12
+      MVN_TEST_OPTIONS: -Dmaven.javadoc.skip=true -DskipTests -Dfast -U -B
+      MVN_COMMON_OPTIONS: -Dflink.forkCount=2 -Dflink.forkCountTestPackage=2
```

Comment:
I'm not sure if this fragmentation really makes things easier to understand.

Reply:
forkCountTestPackage is also not used in the ES poms; thats exclusively used by flink-tests...

Reply:
I'm also not 100% sure yet if this is the best thing to do. We could move them directly to the run command but that makes it one long line too which doesn't do much for readability imho. I did clean-up the not required options

Reply:
For the connection option it makes sense imo since its more infrastructure than anything.